### PR TITLE
Raffle: airtable updates + point logic

### LINKED ISSUE #125: Raffle airtable
- [ ] In airtable, create a form similar to the one from last year but with the new cabin names 
- [ ] Submitting the form as a user (hacker) should update the raffle table to include the new entry 
- [ ] Hook up the front end to airtable so that the front end correctly displays which team is winning and the number of points each team has. 

Look at the logic from last year if you get stuck! 

https://user-images.githubusercontent.com/59738880/212442485-9bb93a63-8eac-411d-95de-a589911336c0.mov

### PR DESCRIPTION
**Pull request by deanframe**
updates airtable form with team names and prize bucket names, and adds point calculation to live site

Fixes #125 

Changelist:

- Gets team point values from airtable
- Points are counted according to [this doc](https://docs.google.com/document/d/19x_MmnKEec5CXrVLLCG6OmUlZ11gD0oT2jQ0hMUwVF4/edit?usp=sharing). There will be two-point event codes and one-point event codes, however the codes aren't finalized yet so I've used placeholder codes defined in `/live-site/lib/data.ts`.
- Duplicate raffle entries aren't counted, and entries without an event code aren't counted (should i make the event code a required question in airtable?)

Notes:

- I'm not the best at writing English 💀 so looking at the airtable form [here](https://airtable.com/app9pkEjk7iSp7mR4/tblPObGGc7FjSMoHI/viwpMQki3ua0UMOOj?blocks=hide) would be much appreciated

Tested:

- locally chrome + firefox on windows 10

### CODE REVIEW

File: live-site/lib/data.ts [hunk 1]
```diff
@@ -10,13 +10,18 @@ const liveSiteTabInfo: TabInfo[] = [
...
 
 const TeamInfo: TeamProps[] = [
-  { name: 'TEAM TATOOINE', points: 100, index: 0 },
-  { name: 'TEAM ESSOS', points: 82, index: 1 },
-  { name: 'TEAM SI WONG', points: 66, index: 2 },
-  { name: 'TEAM ARRAKIS', points: 33, index: 3 },
-  { name: 'TEAM DESSERT BIOME', points: 0, index: 4 }
+  { name: 'Team Tatooine', points: 0, index: 0 },
+  { name: 'Team Essos', points: 0, index: 1 },
+  { name: 'Team SI Wong', points: 0, index: 2 },
```
Comment (jzhzhang66) [inline live-site/lib/data.ts:15]: I think "I" in SI should be lowercase

**Review by jzhzhang66 — APPROVED** [review]
CODE LOOKS 10/10!!!! U truly are the cs king. Left just 1 super minor comment. 

I just had a few questions: 
- Should we test against the case of people submitting multiple times under different names? But like idek how we would test that LOL. Maybe that's just an edge case we'll have to live with haha. 
- Do you know how we pick the raffle winner? Do we just like use a random number generator and then pick whoever is at that index who has submitted a raffle entry? LOL

File: live-site/lib/data.ts (hunk 1 above)
Comment (jzhzhang66) [reply live-site/lib/data.ts:15]: We might have to change that in airtable too since its a tag right?

**Review by karynayen — APPROVED** [review]
LGTM!!
